src/DependencyInjection/Compiler/ReorderTranslationResourceFilesPass.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * @copyright Zicht Online <https://zicht.nl>
+ */
+
+namespace Zicht\Bundle\MessagesBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+class ReorderTranslationResourceFilesPass implements CompilerPassInterface
+{
+    public function process(ContainerBuilder $container)
+    {
+        /* This Compiler Pass is searching for .db translation files to place them at the end
+           This will cause the Message Catalogue to load these resources as last and thus
+           the effect of overriding messages is maintained
+         */
+        $translator = $container->findDefinition('translator.default');
+        $options = $translator->getArgument(4);
+        $resourceFilesByLocale =& $options['resource_files'];
+
+        foreach (array_keys($resourceFilesByLocale) as $locale) {
+            uksort(
+                $resourceFilesByLocale[$locale],
+                function ($keyA, $keyB) use ($resourceFilesByLocale, $locale) {
+                    $fileA = $resourceFilesByLocale[$locale][$keyA];
+                    $fileB = $resourceFilesByLocale[$locale][$keyB];
+                    if (substr($fileA, -3) === '.db' && substr($fileB, -3) !== '.db') {
+                        return 1;
+                    }
+                    if (substr($fileB, -3) === '.db' && substr($fileA, -3) !== '.db') {
+                        return -1;
+                    }
+
+                    return $keyA < $keyB ? -1 : 1;
```

Comment:
Ik had verwacht dat `return substr($resourceFilesByLocale[$locale][$keyB], -3) === '.db' ? -1 : 1` genoeg was geweest.

Dat zou moeten zorgen dat de `.db` bestanden als laatste komen toch?  Just curious...

Reply:
In deze array(s) staan alle files per locale. Dat wil zeggen ook files uit alle bundles of andere locaties.
Ik wilde er wel voor zorgen dat alle overige files in dezelfde volgorde bleven. Anders veroorzaakt dit script wellicht onverwachtse bijwerkingen.

Reply:
Ik denk dat dit niet nodig zou zijn geweest, maar nu het er al staat: :+1:

Reply:
Als ik het eruit liet, dan kreeg ik een array met (de overige) items kris kras doorelkaar

Reply:
Ja, dat had ik ook verwacht, maar ik verwacht dat dat niet uit zou maken.  Maar is prima zo :+1: